roofit/roofitcore/src/RooAddition.cxx

```diff
@@ -173,6 +173,49 @@ void RooAddition::computeBatch(cudaStream_t* stream, double* output, size_t nEve
   dispatch->compute(stream, RooBatchCompute::AddPdf, output, nEvents, pdfs, coefs);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+
+std::string RooAddition::translate(RooFit::Detail::ADContextManager &ctxManager) const
+{
+   // If the number of elements to sum is less than 3, just build a sum expression.
+   // else build a loop to sum over the values.
+   unsigned int eleSize = _set.getSize();
```

Comment:
```suggestion
   std::size_t eleSize = _set.size();
```
Never use `getSize()` of these RooFit collections. We use their STL-style interfaces nowadays.